contracts/examples/hello-world/tests/hello-world.ts

```diff
@@ -1,25 +1,25 @@
-import * as anchor from '@project-serum/anchor';
-import * as fs from 'fs';
-import { Program, BN } from '@project-serum/anchor';
-import { HelloWorld } from '../target/types/hello_world';
+import * as anchor from "@project-serum/anchor";
+import * as fs from "fs";
+import { Program, BN } from "@project-serum/anchor";
+import { HelloWorld } from "../target/types/hello_world";
 
 const CHAINLINK_PROGRAM_ID = "HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny";
 
-describe('hello-world', () => {
-  const provider = anchor.Provider.env();
+describe("hello-world", () => {
```

Comment:
I've noticed many style changes in this PR but looks like we don't have prettier configured in this repository. I suggest we use the one from StarkNet integration and we reformat the codebase one more time: https://github.com/smartcontractkit/chainlink-starknet/blob/develop/.prettierrc.json

Reply:
Let's do that in a follow-up PR since the diff is already quite big

Reply:
https://github.com/smartcontractkit/chainlink-solana/issues/394